index.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+const BbPromise = require('bluebird');
+const _ = require( 'lodash' );
+
+const utils = require('./lib/utils');
+
+module.exports = class ApiGatewayBinaryStatus {
+
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+    this.provider = this.serverless.getProvider('aws');
+    this.stage = this.options.stage;
+
+    Object.assign(this,
+      utils
+    );
+
+    this.hooks = {
+      'after:aws:deploy:finalize:cleanup': this.checkBinarySupportStatus.bind(this),
+    };
+  }
+
+  checkBinarySupportStatus(){
+    this.serverless.cli.log('[serverless-plugin-apig-binary-status] Checking binary support enabled to AWS API Gateway...');
+
+    return BbPromise.resolve()
+      .bind(this)
+      .then(() => this.createTestData(this.stage))
+      .then(() => new BbPromise(resolve => setTimeout(() => resolve(), 65000)))  //replace to depend on serverless-apigwy-binary

```

Comment:
To make sure it runs after deployment of api gateway finished in `serverless-apigwy-binary` https://github.com/ryanmurakami/serverless-apigwy-binary/blob/master/index.js#L71